client/object_replicate.go

```diff
@@ -48,38 +55,38 @@ import (
 //     replicated object;
 //   - [apistatus.ErrContainerNotFound]: the container to which the replicated
 //     object is associated was not found.
-func (c *Client) ReplicateObject(ctx context.Context, id oid.ID, src io.ReadSeeker, signer neofscrypto.Signer) error {
+func (c *Client) ReplicateObject(ctx context.Context, id oid.ID, src io.ReadSeeker, signer neofscrypto.Signer, signedReplication bool) (*neofscrypto.Signature, error) {
```

Comment:
in which cases `signedReplication` gonna be false?

Reply:
For non-initial replication?

Reply:
also as i understand, it will stiil be possible to skip this for some sort of container after https://github.com/nspcc-dev/neofs-api/issues/300, no @roman-khimov?

Reply:
Yes, containers with different consistency policies can have different requirements wrt this as well.